doc/sphinx-guides/source/admin/dashboard.rst

```diff
@@ -0,0 +1,31 @@
+Dashboard
+=========
+
+Dataverse offers a dashboard of administrative tools for superusers only. If you are a logged-in superuser, you will see a "Dashboard" link in the navbar, next to your name. You can verify that you are a superuser by checking your name in the navbar. If it's red, you have the right permissions to use the Dashboard. 
```

Comment:
With the language toggle added to the navbar, the dashboard link has moved from the navbar to the account name dropdown menu. Also, would it be worth linking to how to make a user a superuser so that they could see this page? (Note: the User Administration pg of the Admin Guide is the appropriate page to link to, but that page only mentions "whether or not they have Superuser status" as opposed to clearly pointing out "this is where you grant superuser status". See: http://guides.dataverse.org/en/4.9.2/admin/user-administration.html)

Reply:
Thanks for letting me know about the dropdown menu change, I'll update this! I'll link to the User Admin page too, and look into editing that page to add a little more detail about granting superuser status.